app/assets/javascripts/sephora_style_guide/slider.js

```diff
@@ -0,0 +1,2447 @@
+// Ion.RangeSlider
+// version 2.1.7 Build: 371
```

Comment:
I can see v2.2.0 out on the repo. Want to update that? I am also assuming this was just copy pasted with no customizations made to it. Because it would be hard to update otherwise.